src/widgets/dialogs/LastRunCrashDialog.cpp

```diff
@@ -1,93 +1,162 @@
+    text +=
+        "Crash reports are <b>only stored locally</b> and never uploaded.<br>"
+        u"<br>Please report the crash so it can be prevented in the future."_s;
+
+    if (Modes::instance().isNightly)
+    {
+        text += u" Make sure you're using the latest nightly version!"_s;
+    }
+
+    text +=
+        u"<br>For more information, <a href=\"https://wiki.chatterino.com/Crash%20Analysis/\">consult the wiki</a>."_s;
+#endif
+
+    auto label = layout.emplace<QLabel>(text);
+    label->setTextInteractionFlags(Qt::TextBrowserInteraction);
+    label->setOpenExternalLinks(true);
+    label->setWordWrap(true);
+
+    layout->addSpacing(16);
+
+    auto buttons = layout.emplace<QDialogButtonBox>();
+
+    auto *okButton = buttons->addButton(u"Ok"_s, QDialogButtonBox::AcceptRole);
     QObject::connect(okButton, &QPushButton::clicked, [this] {
         this->accept();
     });
 
-    // Updates
-    //    auto updateUpdateLabel = [update]() mutable {
-    //        auto &updateManager = UpdateManager::instance();
-
-    //        switch (updateManager.getStatus()) {
-    //            case UpdateManager::None: {
-    //                update->setText("Not checking for updates.");
-    //            } break;
-    //            case UpdateManager::Searching: {
-    //                update->setText("Checking for updates...");
-    //            } break;
-    //            case UpdateManager::UpdateAvailable: {
-    //                update->setText("Update available.");
-    //            } break;
-    //            case UpdateManager::NoUpdateAvailable: {
-    //                update->setText("No update abailable.");
-    //            } break;
-    //            case UpdateManager::SearchFailed: {
-    //                update->setText("Error while searching for update.\nEither
-    //                the update service is "
-    //                                "temporarily down or there is an issue
-    //                                with your installation.");
-    //            } break;
-    //            case UpdateManager::Downloading: {
-    //                update->setText(
-    //                    "Downloading the update. Chatterino will close once
-    //                    the download is done.");
-    //            } break;
-    //            case UpdateManager::DownloadFailed: {
-    //                update->setText("Download failed.");
-    //            } break;
-    //            case UpdateManager::WriteFileFailed: {
-    //                update->setText("Writing the update file to the hard drive
-    //                failed.");
-    //            } break;
-    //        }
-    //    };
-
-    //    updateUpdateLabel();
-    //    this->signalHolder_.managedConnect(updateManager.statusUpdated,
-    //    [updateUpdateLabel](auto) mutable {
-    //        postToThread([updateUpdateLabel]() mutable { updateUpdateLabel();
-    //        });
-    //    });
+    if (!getArgs().safeMode)
```

Comment:
I don't think suggesting the use of safe mode when it wouldn't do anything to help is smart